src/main/kotlin/com/jetbrains/micropython/nova/actions.kt

```diff
@@ -170,8 +170,12 @@ class InstantRun : DumbAwareAction() {
     override fun getActionUpdateThread(): ActionUpdateThread = BGT
 
     override fun update(e: AnActionEvent) {
-        e.presentation.isEnabled = e.getData(CommonDataKeys.VIRTUAL_FILE) != null
-
+        val file = e.getData(CommonDataKeys.VIRTUAL_FILE)
```

Comment:
There is no need to separately getting single file and file array.
It's better to get the array and then check if there is exactly one member, ant it is a file.